### ISSUE TITLE
Might have found the issue with Reanimated. Edit: not :)

### ISSUE THREAD

**Issue by hardcodet** [brentvatne/hardcodet#2]
iOS suddenly stopped working too (don't ask me why it worked nicely for more than a day, I don't even want to know), so I did the whole eject and traced things with smaller steps.

The first thing that required changes was reanimated2, and after upgrading, I noticed this in my yarn.lock:

![image](https://user-images.githubusercontent.com/873009/100396255-ca9b0b00-3044-11eb-98b9-2f1fad900e02.png)

Given that I couldn't fetch anymore, that looks fishy enough to investigate further. I really don't understand how this could then have affected my vanilla bare project, but maybe that was just a wrong metro instance serving me content. But it might not be expo after all :)

**Comment by hardcodet** [comment]
Edit:My repro project started breaking right out of the box, only: this time on iOS, while Android was working happily. Sent the project to a friend for testing, and if it's the same for him, I'll open a new issue on expo.

**Comment by brentvatne** [comment]
that's not the correct version of reanimated. see https://docs.expo.io/versions/latest/sdk/reanimated/ for info on how you can use reanimated 2. also, reanimated 2 requires additional setup on ejecting right now. this will change in sdk 40 which will be released soon, but for now you should read the docs for reanimated 2 alpha 6

**Comment by hardcodet** [comment]
Hey there! Thanks for chiming in! 👍 

Ah no, with right out of the box, I mean a clean bare project that's using Reanimated 1. I was using Reaminated 2 in my real app, but I think that one's in the clear now.

I'm a step further and two back:

- on that vanilla bare project, I eventually ran `yarn upgrade` and then fetch magically started to work for it.
- then I switched back to my application, where fetch - without any changes to the code - also started working again, at least on my iOS simulator.
- now I'm back on Android where things still don't work, only that I'm now on my MacBook, where it used to work 24 hours ago.

I assume I'm looking at multiple issues here that mess with me, but I still don't understand them. I hope the repro I sent to my friend wil break there, too, that would be helpful. (I'll know tomorrow, it's nearly 3am here already).

**Comment by hardcodet** [comment]
And now, iOS stopped working as well again. I'm not writing any code, all I'm doing is starting those f... simulators and wonder why stuff keeps breaking non-deterministically :)

**Comment by brentvatne** [comment]
if you can give me a https://stackoverflow.com/help/minimal-reproducible-example i can help, i would suggest working towards that

**Comment by hardcodet** [comment]
I've been trying to get there for the longest time :)

Once I realized that it had to be a caching issue of sorts playing into this, it got easier (still learning a lot, it's my first React Native app).

Cleaning the cache on every restart finally got me reproducible failures, and let me to a Babel setting I introduced with MobX, which reliably messed with the network stack:

```
    ["@babel/plugin-proposal-class-properties", { "loose": false }]
```

Taking the fucker out resolved all issues at once. Thanks for bearing with me an sorry for all the spam. It actually helped just writing down my findings to gather my thoughts (and preventing me from setting my computers on fire ;).
